Migrating-from-v1.md

```diff
@@ -166,6 +166,21 @@ Files.newOutputStream(Path("/your/file.avro")).use { outputStream ->
 }
 ```
 
+## Reading previously written binary encoded (`AvroEncodeFormat.Binary`) files
+
+It was possible to use the `AvroEncodeFormat.Binary` format, which used [binary encoding](https://avro.apache.org/docs/current/spec.html#binary_encoding). The data did not have an embedded schema, so it had to be specified when reading.
+
+```kotlin
+// Previously
+Avro.default.openInputStream(serializer) { decodeFormat = AvroDecodeFormat.Binary(schema) }
+    .from(data).use { avroInputStream -> return avroInputStream.nextOrThrow() }
+
+// Now
+val reader = GenericDatumReader<Any>(schema)
+val decoder = DecoderFactory.get().binaryDecoder(ByteArrayInputStream(data), null)
+Avro.decodeFromGenericData(schema, serializer, reader.read(null, decoder))
+```
```

Comment:
I think you are mixing formats and the avro4k api.

The previous `Binary` format is now handled by `Avro` as it is the pure avro serialization. To encode many elements in "binary", now you need to call as many element as you have the `Avro.encodeToStream` method. To decode, it's the same: loop on `Avro.decodeFromStream` until there is no remaining bytes in the input stream. Side note: you should not encode many times using `Avro` in the same file/stream, as it's unstructured content which needs maintenance, version control, and mainly the schema! The `AvroObjectContainer` spec has been created exactly for that purpose.

The previous `Data` format is now handled by `AvroObjectContainer` which is the official name in the specs.

Btw, all the avro serializations... Are binary, that's why this concept has been removed in the v2.